modules/local/bigstitcher/module/main.nf

```diff
@@ -0,0 +1,71 @@
+process BIGSTITCHER_MODULE {
+    tag { meta.id }
+    container { task.ext.container ?: 'ghcr.io/janeliascicomp/bigstitcher:2.4.1-spark3.2.1-jdk8' }
+    cpus { spark.driver_cores }
+    memory { spark.driver_memory }
+
+    input:
+    tuple val(meta), path(bigstitcher_container), val(spark)
```

Comment:
I think the identifier "bigstitcher_container" is very confusing, because it conflicts with the Docker "container" used above. Names like "output_path" and "output_uri" might be better.